README.md

```diff
@@ -80,10 +80,10 @@ Now, copy the compiled application artifacts `build/install/{project_name}` from
 All of the described services should be packaged inside their own Docker
 container and orchestrated via Docker Compose.
 Therefore, adapt the existing `docker-compose.yml` in the root project folder.
-Make sure to set appropriate dependencies to allow an automatic bootstrapping of the service composition via `docker-compose up`.
+Make sure to set appropriate dependencies to allow an automatic bootstrapping of the service composition via `docker compose up`.
```

Comment:
Hi Marcel, geht das mittlerweile auch standardmäßig auf Linux? Damals ging das ja nur auf Windows mit Docker Desktop.

Reply:
Hi Stefan, ja das ist mittlerweile der präferierte Weg in den Docker Guidelines. In Linux muss ggf. noch das ` `docker-compose-plugin` installiert werden, was derselbe Aufwand ist wie `docker-compose` zu installieren.

Reply:
Das sollten wir ggf. dann am Anfang dokumentieren denke ich, damit das self-contained ist. Merge Rechte hast du?!